examples/basic/media-transcoder.rb

```diff
@@ -0,0 +1,16 @@
+class MediaTranscoder
+  def transloadit_client
+    unless @transloadit
+      @transloadit = Transloadit.new({
+        key: ENV.fetch('TRANSLOADIT_KEY'),
+        secret: ENV.fetch('TRANSLOADIT_SECRET')
+      })
+    end
+    @transloadit
+  end
+
+  def get_status!(assembly_id)
+    req = Transloadit::Request.new('/assemblies/' + assembly_id.to_s, ENV.fetch('TRANSLOADIT_SECRET'))
+    req.get.extend!(Transloadit::Response::Assembly)
+  end
+end
```

Comment:
Nitpick: if you can, please add newlines to the end of files, thanks!

Depending on your editor, that might be just a setting to have that done automatically.